lesson-7/server/api.js

```diff
@@ -0,0 +1,56 @@
+const MongoClient = require('mongodb').MongoClient;
+const config = require('./config');
+
+function executeQuery(query) {
+    return new Promise((resolve, reject) => {
+        MongoClient.connect(config.MONGO_URL).then(database => {
+            var todos = query(database)
```

Comment:
Nested Promises? Let's flatten it

Reply:
needed to make sure the connection is closed after the query... what different approach could be used?